libs/execution_context/ebpf_maps.c

```diff
@@ -1140,6 +1249,7 @@ _create_lru_hash_map(
 {
     ebpf_result_t retval = EBPF_SUCCESS;
     ebpf_core_lru_map_t* lru_map = NULL;
+    uint32_t partition_count = min(ebpf_get_cpu_count(), EBPF_LRU_MAXIMUM_PARTITIONS);
```

Comment:
okay, here the partition count gets the min between CPU cores and max partitions 8.

I have seen in VMs with 40 cores (vcpu). Will there be still contention? 

Use-case: In Server Container Networking.

Reply:
This is a tradeoff between memory footprint and performance. If we have data that this is an issue we can try raising it.